BitcoinKit/BitcoinKitPrivate.h

```diff
@@ -18,7 +18,7 @@ NS_ASSUME_NONNULL_BEGIN
 @end
 
 @interface _Key : NSObject
-
++ (NSData *)curveOrderMinus:(NSInteger)smallNumber;
```

Comment:
@kishikawakatsumi we probably do not want this in the `BitcoinKitPrivate.h` file? But I was unsure how to create an Objective C `.m`/`.h` file tuple in the Tests and refer to OpenSSL. I created an `.m` file, a bridging header and added Library Search paths to include openssl but didn't get it to work.

Reply:
Thanks.
I'm not sure about how to include C file in the Tests.
But `curveOrderMinus` is not suitable to be included in `BitcoinKitPrivate.h `.